src/loaders/ObjectLoader.js

```diff
@@ -796,7 +875,35 @@ Object.assign( ObjectLoader.prototype, {
 
 		};
 
-	}()
+	}(),
+
+	bindSkeletons: function ( object, skeletons ) {
+
+		if ( Object.keys( skeletons ).length === 0 ) return;
+
+		object.traverse( function ( obj ) {
+
+			if ( obj.isSkinnedMesh === true && obj.skeletonUUID !== undefined ) {
+
+				var skeleton = skeletons[ obj.skeletonUUID ];
+
+				if ( skeleton === undefined ) {
+
+					console.warn( 'THREE.ObjectLoader: Not found Skeleton whose uuid is ' + obj.sjeletonUUID );
```

Comment:
Sorry if im being a bit annoying :), i really want this PR to be merged i was comparing it to what i was doing on nunu xD 
Awesome work thanks!